tests/testthat/test-spatial_clustering_cv.R

```diff
@@ -141,6 +140,55 @@ test_that("using sf", {
 
 })
 
+test_that("using custom functions", {
+
+  custom_cluster <- function(dists, v, ...) {
+    clusters <- kmeans(dists, centers = v, ...)
+    letters[clusters$cluster]
+  }
+
+  set.seed(11)
+  rs1 <- spatial_clustering_cv(Smithsonian,
+                               coords = c(latitude, longitude),
+                               v = 2
+  )
```

Comment:
In cases like this, the tidy style guide suggests putting each argument on each own line and indenting a bit differently: 
https://style.tidyverse.org/syntax.html#long-lines